docs/Actions.md

```diff
@@ -1470,6 +1470,7 @@ mailgun(
   postmaster: "MY_POSTMASTER",
   apikey: "MY_API_KEY",
   to: "DESTINATION_EMAIL",
+  from: "My awesome app",
```

Comment:
Can you make the example here a little bit more generic? like EMAIL_FROM_NAME or something? "My awesome app" doesn't provide enough context for the `from` value for users reading this documentation.